x-pack/metricbeat/modules.d/sql.yml.disabled

```diff
@@ -9,4 +9,5 @@
 
   driver: "postgres"
   sql_query: "select now()"
+  sql_response_format: table
```

Comment:
This line should be added to `x-pack/metricbeat/module/sql/_meta/config.yml`, and then run `mage fmt update` so this file gets automatically updated.

Reply:
This should fix the issue in CI.